packages/go-expoitdb/PKGBUILD

```
@@ -0,0 +1,43 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=go-exploitdb
+pkgver=159.7d2fc13
+pkgrel=1
+pkgdesc='Tool for searching Exploits from Exploit Databases, etc.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-exploit')
+url='https://github.com/vulsio/go-exploitdb'
+license=('MIT')
+makedepends=('git' 'go')
+source=("git+https://github.com/vulsio/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
```

```suggestion
pkgver() {
  cd $pkgname

  ( set -o pipefail
    git describe --long --tags --abbrev=7 2>/dev/null |
      sed 's/\([^-]*-g\)/r\1/;s/-/./g' ||
    printf "%s.%s" "$(git rev-list --count HEAD)" \
      "$(git rev-parse --short=7 HEAD)"
  )
}
```

cf. https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git